dm/cmd/dm-syncer/config.go

```diff
@@ -168,6 +168,7 @@ func (c *commonConfig) newSubTaskConfig(args []string) (*config.SubTaskConfig, e
 	fs.StringVar(&cfg.LogRotate, "log-rotate", "day", "log file rotate type, hour/day")
 	fs.BoolVar(&cfg.EnableGTID, "enable-gtid", false, "enable gtid mode")
 	fs.BoolVar(&cfg.SafeMode, "safe-mode", false, "enable safe mode to make syncer reentrant")
+	fs.BoolVar(&cfg.DisableSafeModeResume, "disable-safe-mode-resume", false, "disable safe mode to resume ")
```

Comment:
Why must this command-line argument be added?

Reply:
Sry, it was a mistake. What we need do is that just add it to task config file.

I will remove it.